multinode/roles/monitoring_install/tasks/main.yml

```
@@ -0,0 +1,149 @@
+  when: "inventory_hostname == groups['validators'][item]"   
+
+- name: Change node_exporter permissions
+  become: true
+  shell: chmod +x /usr/local/bin/promtail    
+
+- name: Start the promtail service
+  become: true
+  systemd:
+    name: promtail
+    state: started
+    enabled: yes   
```

yamllint

[error] trailing spaces

(trailing-spaces)